src/web/components/Flex.tsx

```diff
@@ -17,6 +17,9 @@ export const Flex = ({
             display: flex;
             flex-direction: ${direction};
             justify-content: ${justify};
+            /* Fixes IE 10/11 bug that collapses this container by default: */
+            /* stylelint-disable-next-line property-no-vendor-prefix */
+            -ms-flex-positive: 1;
```

Comment:
Not sure it makes any difference here, but is it generally good to have the browser specific prefixed properties first so that standards ones override it in browsers that understand both?

Reply:
Emotion compiles down CSS to support all browsers, I am unsure how it handles flexbox but I do know that IE does not understand `flex: 1` but instead needs `flex: 1 1 auto`

Reply:
@jfsoul I don't think the order matters in this case, because we are only using the prefix to instruct specific behaviour for IE rather than to make a general rule that should apply across all browsers, if that makes sense

@liywjl I don't think that's true in this case. The compiled CSS only has "-ms-flex-positive" in it, not "flex-grow" for other browsers, so it is ignored in the case of other browsers, which is the desired behaviour.